Source/Scenes/Onboarding/1_AnalyticsPermission/AnalyticsPermission.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AnalyticsPermission.swift
+//  Zupreme
+//
+//  Created by Andrei Radulescu on 09/11/2018.
+//  Copyright © 2018 Open Zesame. All rights reserved.
+//
+
+import Foundation
+
+private typealias € = L10n.Scene.AnalyticsPermission
+
```

Comment:
I have an idea that all Scenes should be named using a verb. Like `Send`, `Receive`, `CreateNewWallet`, `ChoseWallet` - when possible. For `Settings` it was trickier, thus I just named it `Settings` (if you can come up with a better name, please let me know). Anyway, maybe this can be named `AskUserForPermissionToTrackEvents` or `AskForAnalyticsPermissions`? Or so? Hmm maybe we should renamed `TermsOfService` to `AskUserToAcceptTermsOfService`?

Reply:
Sure, sounds good. We also need to rename WarningERC20 besides TermsOfService. AskUserFor prefix is good enough let’s open a separate issue for renaming the Scenes besides Analytics.

Reply:
great!